client/src/App.jsx

```diff
@@ -2,9 +2,20 @@ import React from 'react';
 import './App.css';
 
 import Navbar from './components/Navbar/Navbar';
+import Slider from './components/Slider/Slider';
 
 const App = () => {
-  return <Navbar />;
+  return (
+    <React.Fragment>
+      <Navbar />
+      <Slider
+        slides={[
+          'http://localhost:8080/asset/welcome-to-uni.jpg',
```

Comment:
Nigdy nie wpisuj na sztywno URLa.
Jak będziesz to chciał odpalić na innym środowisku lub spublikować to z automatu nie zadziała
Najlepiej ścieżka relacyjna.
Gorzej: URL w environments